python/chronos/src/bigdl/chronos/forecaster/autoformer_forecaster.py

```diff
@@ -49,6 +49,7 @@ def __init__(self,
                  optimizer="Adam",
                  loss="mse",
                  lr=0.0001,
+                 lr_scheduler_milestones=[3, 4, 5, 6, 7, 8, 9, 10],
                  metrics=["mse"],
                  seed=None,
                  distributed=False,
```

Comment:
add api doc (i.e. `:param lr_scheduler_milestones: xxxx`) for this parameter

Reply:
done